### ISSUE TITLE
Glue

### ISSUE THREAD

**Issue by alikeobservant** [studio-minus/ppg-bugs-and-requests#1221]
**Feature description**
You can put glue on an object. When this glued object makes contact with another object, it will be glued together. Maybe varying levels of glue strength?

**Feature purpose**
Would open the door to self assembling machines and much more

**Comment by ppgplayer** [comment]
I agree. As a glue eater, this will be valuable for personal reasons.

**Comment by Drogo-man** [comment]
Fixed cable or metal binding.

**Comment by alikeobservant** [comment]
> 

You have to manually put cables and binding. I'm talking if it just touches the object it will stick.

**Comment by Drogo-man** [comment]
> > 
> 
> You have to manually put cables and binding. I'm talking if it just touches the object it will stick.

You can freeze time and put them together.

**Comment by ppgplayer** [comment]
> > > 
> > 
> > 
> > You have to manually put cables and binding. I'm talking if it just touches the object it will stick.
> 
> You can freeze time and put them together.

Mate, you are completely missing the point. He wants something that will STICK when an object TOUCHES THE GLUE. Your solution not only does not work, but also doesn't even come close.
